lib/cucumber/runtime.rb

```diff
@@ -79,7 +79,7 @@ def run!
 
       receiver = Test::Runner.new(@configuration.event_bus)
       compile features, receiver, filters, @configuration.event_bus
-      @configuration.notify :test_run_finished
+      @configuration.notify :test_run_finished, !failure?
```

Comment:
To avoid setting public in private can we just move this up to above private?

Reply:
Don't get that comment, sorry 🤔

Reply:
This method you're using (I think it's a method), is a hidden public/private method (As in it's actually public, but nestled way down deep inside the private methods, then toggled back to public).

Ideally as we're using it, lets fix it up and just make it public in the normal way.

Reply:
Got it, thanks 👌